tests/foreman/cli/test_capsule.py

```diff
@@ -0,0 +1,157 @@
+# -*- encoding: utf-8 -*-
+# vim: ts=4 sw=4 expandtab ai
```

Comment:
Is there a good reason to set the `autoindent` option? There are several indentation modes, and overriding user preferences seems like a bad idea.